conda-test-requirements.txt

```diff
@@ -4,7 +4,7 @@ pytz
 toolz
 
 # Test Framework
-hypothesis
+hypothesis<5.2.0
```

Comment:
please create a follow-up ticket to upgrade hypothesis

Reply:
Done: https://github.com/JDASoftwareGroup/kartothek/issues/205